dkpro-core-io-nyt-asl/src/main/resources/desc/type/NYTArticleMetaData.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<typeSystemDescription xmlns="http://uima.apache.org/resourceSpecifier">
+	<name>NYTArticleMetaData</name>
+	<description />
+	<version>1.0</version>
+	<vendor />
+	<types>
+		<typeDescription>
+			<name>de.tudarmstadt.ukp.argumentext.demonstrator.metadata.NYTArticleMetaData
+			</name>
+			<description>A document annotation that describes the metadata of a
+				newspaper article.
+			</description>
+			<supertypeName>uima.tcas.DocumentAnnotation</supertypeName>
```

Comment:
There can only be one DocumentAnnotation per CAS and that must be a DocumentMetaData annotation in the case of DKPro Core. This annotation should instead inherit from AnnotationBase.

Actually, even better would be to use MetaDataStringField annotations from the `metadata` module in DKPro Core.